src/main.rs

```diff
@@ -15,8 +19,11 @@ use gst::BinExt;
 use std::cell::RefCell;
 use std::env::args;
 use std::error;
+use std::path::{Path, PathBuf};
 use std::rc::{Rc, Weak};
 
+const SETTINGS_FILE: &'static str = "settings.toml";
```

Comment:
Should be stored in `glib::get_user_config_dir()/application_name/settings.toml` :)

Reply:
I discover things haha.